e2e/cypress/integration/dataset.spec.js

```diff
@@ -66,7 +67,7 @@ describe('Harvest Dataset Validation', () => {
     })
 
     it('validates data json metadata', () => {
-        cy.visit('/dataset/series-of-aerial-images-over-monte-vista-national-wildlife-refuge-acquired-in-1960')
+        cy.visit('/dataset/series-of-aerial-images-over-alamosa-national-wildlife-refuge-acquired-november-1941')
```

Comment:
Not a blocker, but should consider "finding" a dataset in real time by visiting harvest, organization, or main dataset page instead of hardcoding dataset title as that may change in the future.